packages/utils/src/getLoginHistoryFormattedData.tsx

```diff
@@ -0,0 +1,56 @@
+import moment from 'moment';
+import Bowser from 'bowser';
+import { LoginHistory } from '@deriv/api-types';
+
+type TData = {
+    id: number;
+    date: string;
+    action: string;
+    browser: string;
+    ip: string;
+    status: string;
+}[];
+
+type TUserAgent =
+    | (Bowser.Parser.Details & {
+          app?: string;
+      })
+    | undefined;
+
+/** Format API login_history value to a better ones */
+const getLoginHistoryFormattedData = (login_history: LoginHistory) => {
```

Comment:
The scope of the card is TS migration. Is creation of this function Not falling out of the scope of this PR?  :)
TS migration implies migration of JS files to TS, not creation of new things. Only some refactoring is normally done together with TS migration if needed.
Also, SonarCloud points at regex here as a security hotspot (not vulnerability though). Can the regex be improved?

Reply:
Going forward we can ensure tasks other than TS migration is not included in the PR. 👍🏼 
regarding the regex, i think the regex was copied from backend and also, in real scenario, the string won't be long enough to cause linear runtime complexity during regex search.